kingfisher_scrapy/spiders/ghana.py

```diff
@@ -18,30 +13,9 @@ class Ghana(CompressedFileSpider):
             Download only data until this month (YYYY-MM format).
             If ``from_date`` is provided, defaults to the current month.
     """
-
     name = 'ghana'
 
     # BaseSpider
-    date_format = 'year-month'
     default_from_date = '2019-07'
 
-    # SimpleSpider
-    data_type = 'record_package'
-
-    def start_requests(self):
-        yield scrapy.Request(
-            'https://www.ghaneps.gov.gh/ocds/services/recordpackage/getrecordpackagelist',
-            meta={'file_name': 'list.json'},
-            callback=self.parse_list,
-        )
-
-    @handle_http_error
-    def parse_list(self, response):
-        urls = response.json()['packagesPerMonth']
-        for url in reversed(urls):
-            if self.from_date and self.until_date:
-                year, month = map(int, url.rsplit('/', 2)[1:])
-                url_date = datetime.datetime(year, month, 1)
-                if not (self.from_date <= url_date <= self.until_date):
-                    continue
-            yield self.build_request(url, formatter=join(components(-2), extension='zip'))
+    base_url = 'https://www.ghaneps.gov.gh'
```

Comment:
```suggestion
    # EuropeanDynamicsBase
    base_url = 'https://www.ghaneps.gov.gh'
```

And the same for the others